src/metabase/api/setup.clj

```diff
@@ -86,4 +88,108 @@
       (catch Throwable e
         (response-invalid :general (.getMessage e))))))
 
+
+;;; Admin Checklist
+
+(defn- admin-checklist-values []
+  (let [has-dbs?           (db/exists? Database)
+        has-dashboards?    (db/exists? 'Dashboard)
+        has-pulses?        (db/exists? 'Pulse)
+        has-labels?        (db/exists? 'Label)
+        has-hidden-tables? (db/exists? 'Table, :visibility_type [:not= nil])
+        has-metrics?       (db/exists? 'Metric)
+        has-segments?      (db/exists? 'Segment)
+        num-tables         (db/select-one-count 'Table)
+        num-cards          (db/select-one-count 'Card)
+        num-users          (db/select-one-count 'User)]
+    [{:title       "Add a database"
+      :group       "Get connected"
+      :description "TODO - Write something good here"
+      :link        "/admin/databases/create"
+      :completed   has-dbs?
+      :triggered   :always}
```

Comment:
this is the approach that @salsakran recommended -- everything in a single place